public/app/features/api-keys/ApiKeysPage.tsx

```diff
@@ -146,6 +146,13 @@ export class ApiKeysPageUnconnected extends PureComponent<Props, State> {
               const showTable = apiKeysCount > 0;
               return (
                 <>
+                  {config.featureToggles.serviceAccounts && (
+                    <Alert title="Switch from API keys to Service accounts" severity="info">
+                      Service accounts give you more control. API keys will be automatically migrated into tokens inside
+                      respective service accounts. The current API keys will still work, but will be called tokens and
+                      you will find them in the detail view of a respective service account.
```

Comment:
maybe a link to our documentation?

Could you send a pic on how this looks? Alert seems like it warns the user, but unsure how that looks for us

Reply:
I think we hide this banner until we have migration implemented. So we will show it when migration is not happened.